cmd/habitatctl/commands/helpers.go

```diff
@@ -27,5 +27,5 @@ func sendRequest(command string, args []string) {
 	if err != nil {
 		fmt.Printf("Error: couldn't read response from habitat service: %s\n", err)
 	}
-	fmt.Println(res)
```

Comment:
This Println is what echoes the response to the command line. If we move `SendRequest` to the client package, we should make a wrapper for it in habitatctl that makes sure the output gets printed that all the commands can use